src/components/components/RuiTimePicker.vue

```diff
@@ -6,12 +6,14 @@ type TimePickerEditMode = 'hour' | 'minute' | 'second' | 'millisecond';
 
 export interface RuiTimePickerProps {
   accuracy?: TimeAccuracy;
+  borderless?: boolean;
 }
 
-const modelValue = defineModel<Date>({ required: true });
+const modelValue = defineModel<Date>({ required: false, default: new Date(2024, 0, 1) });
```

Comment:
Is there any specific reason we are changing the v-model to be optional here?

Reply:
Ah sorry, I forgot to change this part.
Yeah what I want to achieved is the point (2), because sometimes it can be undefined
Should be updated now @kelsos